.github/workflows/release-ts-api-package.yml

```diff
@@ -1,16 +1,55 @@
-name: Release Ts API package
+# Reference: ci.yml
+name: Release Ts API Package
+
 on:
   workflow_dispatch:
-
+      inputs:
+        tag:
+          description: 'Publish tag'
+          required: true
+          default: 'latest'
 env:
   NODE_AUTH_TOKEN: ${{ secrets.RELEASE_TS_API_PACKAGE_TOKEN }}
 
 jobs:
-  publish:
+        
+  update-publish:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout Code
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v4
+
+      - name: Set env variables
+        run: echo "RELEASE_IMAGE_VERSION=${{ inputs.tag }}" >> "$GITHUB_ENV"
+
+      - name: Pull litentry image optionally
+        run: |
+          docker pull parity/polkadot
+          docker pull litentry/litentry-worker:${{ inputs.tag }}
+          docker pull litentry/litentry-cli:${{ inputs.tag }}
+          docker pull litentry/litentry-parachain:${{ inputs.tag }}
```

Comment:
It seems to require that all these tags are in sync.
I'm not sure if that's always the case

cc @grumpygreenguy